.github/workflows/build.yaml

```
@@ -50,11 +50,11 @@ jobs:
         # Can be changed back to npm@7 when https://github.com/npm/cli/issues/3637 is resolved
         run: npm install -g npm@7.20.2
 
-      - name: Install verdaccio
-        run: npm install -g verdaccio
-
       - name: Install Dependencies
-        run: npm install --force
+        run: npm ci --force
+
+      - name: Bootstrap
+        run: npx lerna run bootstrap --stream
```

I think these two separate steps can be one now if we are changing things here anyway (also we should probably  not use force, there was a weird win issue, but I'd try it and see if it all works without it)

```suggestion
        run: npm run bootstrap-ci
```

---

Is it ok to don't have `--force`?

---

I think so, I think we were running into a weird windows issue that is gone now, but you'll have to try 😄